morango/models.py

```diff
@@ -100,10 +96,33 @@ class AbstractStoreModel(models.Model):
     last_saved_instance = models.UUIDField()
     last_saved_counter = models.IntegerField()
     last_saved_counter_per_instance = models.TextField(default="{}")  # RMC
+    model_name = models.CharField(max_length=40)
+
+    class Meta:
+        abstract = True
+
+
+class AbstractDatabaseMaxCounter(models.Model):
+
+    instance_id = models.UUIDField()
+    max_counter = models.IntegerField()
+    transfer_session_id = UUIDField()
+    kind = models.BooleanField()
 
     class Meta:
         abstract = True
 
+    @classmethod
+    def filter_max_counters(cls, query):
```

Comment:
A clearer name (so it doesn't look like it's just a queryset-style filter) might be `get_max_counters_for_filter`. And let's call the argument "filter" rather than "query".